src/common.ts

```diff
@@ -106,7 +106,7 @@ export const packSDJWT: PackSDJWT = async (claims, disclosureFrame, hasher, opti
       }
     }
 
-    const decoys = createDecoy(disclosureFrame[SD_DECOY_COUNT], hasher);
+    const decoys = createDecoy(disclosureFrame[SD_DECOY_COUNT], hasher, options?.generateSalt);
```

Comment:
@ragnika  is it possible that `generateSalt` is not provided?
I see that it is required for `issueSDJWT`, shouldn't it follow the same logic here?

Reply:
@linasi `issueSDJWT` shouldn't require the generateSalt, I'll update the type to remove that.
It was in the options arg for that reason.